deploy/kubedirector/kubedirector.hpe.com_kubedirectorclusters_crd.yaml

```diff
@@ -147,6 +147,9 @@ spec:
                           ephemeral-storage:
                             type: string
                             pattern: '^([0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'
+                      serviceAccountName:
```

Comment:
I think this is in the wrong place... you've got it as part of "resources" rather than part of the role spec.